src/status_im/ui/screens/wallet/send/subs.cljs

```diff
@@ -75,25 +75,40 @@
                        send-transaction)
                      edit)))
 
+(defn check-sufficient-funds [transaction balance symbol amount]
```

Comment:
It'd be good to have some tests for `check-sufficient-funds` & ` check-sufficient-gas` documenting the behavior, `wallet.send/transactions` & `wallet.send/unsigned-transaction` could also be extracted and tested